### PR TITLE
Kbadmin tests

### PR DESCRIPTION
**Pull request by aditya-matukumalli**

### CODE REVIEW

File: test/bigtest/tests/jobs-filter-test.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      await jobs.runningStatusCheckbox.clickInProgressJobCheckbox();
+      await jobs.whenInstancesArePresent(totalJobsAmount);
+    });
+
+    describe('filtering by', () => {
+      beforeEach(async function () {
+        await jobs.resultCheckbox.clickSuccessResultCheckbox();
+      });
+
+      it('success filter should show all successful jobs', () => {
```
Comment (doytch) [inline test/bigtest/tests/jobs-filter-test.js]: This case isn't passing for me with `AssertionError: expected 13 to equal 2`.

I wonder whether lines 73-76 should have different values for `value` and `label`? At the very least, `label` in L73 seems like it should be `Success` and not `success`.

File: test/bigtest/tests/jobs-view-test.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    it('should render info logs accordion', () => {
+      expect(jobsView.isInfoLogsAccordionPresent).to.be.true;
+    });
+
+    it('should render the delete button in the action dropdown', () => {
+      expect(jobsView.headerDropdownMenu.isDeleteButtonPresent).to.be.true;
+    });
+  });
+
+  describe('click delete job', () => {
```
Comment (doytch) [inline test/bigtest/tests/jobs-view-test.js]: I assume this block and other deletion-related blocks require the work from ERM-290, right? Cause these three describe's are failing for me atm.

File: test/bigtest/network/factories/job.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  dateCreated: () => faker.date.recent().toISOString(),
+  name: () => faker.random.words(),
+  ended: () => faker.date.recent().getTime(),
+  source: () => faker.random.arrayElement(['Harvester']),
+  started: () => faker.date.recent().getTime(),
+  status: () => {
+    const val = faker.random.arrayElement(['Queued', 'In progress', 'Ended']);
+    return {
+      id: () => faker.random.uuid(),
+      value: val,
```
[the remark this answers is not in the document]
Comment (aditya-matukumalli) [reply test/bigtest/network/factories/job.js]: not need I think as we use just the _label_ here for our filtering. also`id` doesn't make a difference here as we are pulling them from the `job` factory directly

File: test/bigtest/tests/jobs-filter-test.js [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+import setupApplication from '../helpers/setup-application';
+import JobsInteractor from '../interactors/jobs';
+
+describe('Local kb admin Filters', () => {
+  setupApplication();
+  const jobs = new JobsInteractor();
+
+  describe('status filter tests', () => {
+    const endedJobsAmount = 4;
```
[the remark this answers is not in the document]
Comment (aditya-matukumalli) [reply test/bigtest/tests/jobs-filter-test.js]: We use the `endedJobsAmount` to create as many ended jobs

File: test/bigtest/tests/jobs-view-test.js (hunk 2 above)
Comment (aditya-matukumalli) [reply test/bigtest/tests/jobs-view-test.js]: Yes

File: test/bigtest/tests/jobs-filter-test.js [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+      it('success filter should show all successful jobs', () => {
+        expect(jobs.instanceList.size).to.equal(successJobsAmount);
+      });
+
+      describe('partialy successful jobs', () => {
+        beforeEach(async function () {
+          await jobs.clickResetAll();
+        });
+
+        describe('should', () => {
```
[the remark this answers is not in the document]
Comment (aditya-matukumalli) [reply test/bigtest/tests/jobs-filter-test.js]: Yeah, seems to work when I merge both the beforeEaches now. It wasn't working previously.

**Comment by doytch** [comment]
Added `do-not-merge` label until https://github.com/folio-org/stripes-components/pull/1011 is merged.

File: Jenkinsfile [hunk 6]
```diff
@@ -6,4 +6,5 @@ buildNPM {
   runLint = 'yes'
   runSonarqube = 'no'
   runTest = 'no'
```
Comment (doytch) [inline Jenkinsfile]:
```suggestion
  runTest = 'yes'
```

**Pull request merged**